S2VX.Game/SongSelection/SongSelectionScreen.cs

```diff
@@ -49,6 +52,8 @@ public class SongSelectionScreen : Screen {
 
         private bool DirectoryContainsDirectories(string dir) => Storage.GetDirectories(dir).Any();
 
+        public override void OnEntering(IScreen last) => Cursor.Reset();
```

Comment:
This doesn't get called when exiting from the EditorScreen/GameScreen or `OnExiting` from another SongSelectionScreen. The cursor is only reset when navigating into another `SongSelectionScreen`

Reply:
I'll fix this.